ledger/participant-integration-api/src/test/lib/scala/platform/store/dao/JdbcLedgerDaoPostCommitValidationSpec.scala

```diff
@@ -148,7 +149,8 @@ private[dao] trait JdbcLedgerDaoPostCommitValidationSpec extends LoneElement {
     }
   }
 
-  it should "refuse to insert entries with conflicting transaction ids" in {
+  // TODO figure out what is leading to exception here (maybe the unique constraint on the event_id?), then figure out whether we can re-add it, or if we need this feature at all?
```

Comment:
@gerolf-da do you know why we want the IndexDB to reject this case? It should not happen if upstream works correctly, and if it does not, we wouldn't want to stop ingesting, but issue a warning.

Reply:
We do not have a good story for handling incorrect data from the ReadService. The current index DB schema has some uniqueness constraints that prevent things like duplicate contract keys. These likely were inherited from sandbox classic times and never revisited.

Like you said, we should look over all index/indexer operations and come up with a design for how to handle corrupt upstream data on a best-effort basis.